File: pset6/mario/mario.py
from cs50 import get_int
while(True):
    length = get_int("Enter length")
    if length in range(1, 9, 1):
        break
for i in range(length):
    for j in range(length - i - 1):
        print(" ", end="")
    for i2 in range(i + 1):
        print("#", end="")
    print("  ", end="")
    for i3 in range(i + 1):
        print("#", end="")
    print("")
    
    
    
    

File: pset1/mario/mario.c

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int length = 0;
    do
    {
        length = get_int("Enter the length!");
    }
    while (length < 1 || length > 8);
    for (int i = 0; i < length ; i++)
    {
        for (int j = 1 ; j < length - i ; j++)
        {
            printf(" ");
        }
        for (int i2 = 0 ; i2 <= i ; i2++)// to make the left side
        {
            printf("#");
        }
        printf("  ");
        for (int i3 = 0; i3 <= i; i3++)//to make the right side
        {
            printf("#");
        }


        printf("\n");


    }

}